stage1/init/common/kvm_mount.go

```diff
@@ -81,13 +81,14 @@ func MountSharedVolumes(root string, appName types.ACName, volumes []types.Volum
 		}
 
 		destination := filepath.Join(appRootfs, mntPath)
-		// TODO: verify if rkt should do this, or it should be some outer responsibility
-		// to ensure if such patch exists
-		if err := ensureDestinationExists(source, destination); err != nil {
+		// TODO: check if symlinked source tries to escape root
+		// TODO: verify if rkt should ensure existance of destination there, or it should be some
+		// outer responsibility to ensure if such patch exists
```

Comment:
s/patch/path

I agree, as you said, it would be nice to have a test for that :)